ui/components/Header.js

```diff
@@ -176,13 +176,9 @@ class Header extends React.Component {
                     <MesheryNotification />
                   </div>
 
-                  {/* <Tooltip title="MeshSync Status">
-                    <div style={{ padding : "1rem", height : "2rem", width : "2rem", borderRadius : "50%", backgroundColor : this.state.meshSyncStatus.status === "ENABLED" ? "green" : "red" }} />
-                  </Tooltip>
-
-                  <Tooltip title="Broker Status">
-                    <div style={{ padding : "1rem", height : "2rem", width : "2rem", borderRadius : "50%", backgroundColor : this.state.brokerStatus ? "green" : "red" }} />
-                  </Tooltip> */}
```

Comment:
Are we not showing the broker status logo as well? It's different from MeshSync

Reply:
Right - a different PR. 👍